services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -3339,6 +3338,30 @@ private OfflinePushStatusInfo getOffLineJobStatus(
               Optional.of(currentReturnStatusDetails.orElse("") + "Parent Kafka topic truncated");
         }
       }
+
+      // Update version status on parent
+      // COMPLETED -> ONLINE
+      // ERROR -> ERROR
+      // ERROR in some region but COMPLETED in others -> PARTIALLY_ONLINE
+      if (!incrementalPushVersion.isPresent()) {
+        final VersionStatus finalVersionStatusOnParent;
+        if (currentReturnStatus.equals(ExecutionStatus.COMPLETED)) {
+          finalVersionStatusOnParent = VersionStatus.ONLINE;
+        } else if (currentReturnStatus.equals(ExecutionStatus.ERROR) && successCount > 0) {
+          finalVersionStatusOnParent = VersionStatus.PARTIALLY_ONLINE;
```

Comment:
For Nuage, if the final status is `PARTIALLY_ONLINE`, do we need to show the status for each individual colo?

Reply:
We already return `coloToCurrentVersions` in the response